File: C/AngleBetweenHandsOfClock.c
#include <stdio.h>
#include <math.h>

double angleClock(int hour, int minutes) {
    double hourDegree = (hour % 12) * 30 + 0.5 * minutes;
    double minuteDegree = minutes * 6;

    double diff = fabs(hourDegree - minuteDegree);
    return fmin(diff, 360 - diff);
}

int main() {
    int hour, minutes;

    printf("Enter hour and minutes: ");
    scanf("%d %d", &hour, &minutes);

    printf("Angle: %.1f\n", angleClock(hour, minutes));

    return 0;
}

File: Python/AnglebetweenhandsofClock.py
class Solution(object):
    def angleClock(self, hour, minutes):
        hourDegree = (hour % 12) * 30 + 0.5 * minutes
        minuteDegree = minutes * 6

        diff = abs(hourDegree - minuteDegree)
        return min(diff, 360 - diff)

if __name__ == "__main__":
    hour = int(input("Enter hour: "))
    minutes = int(input("Enter minutes: "))

    sol = Solution()
    print("Angle:", sol.angleClock(hour, minutes))